File: Python/004_Program/program.py
#!/usr/bin/env python3
# File Copy - Copy files
import shutil

def main():
    print("=== File Copy ===")
    source = input("Source file: ")
    dest = input("Destination: ")
    
    try:
        shutil.copy2(source, dest)
        print("File copied successfully!")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    main()

File: CSharp/004_Program/Program.cs
// File Copy Tool
using System;
using System.IO;
class Program {
    static void Main() {
        Console.Write("Source: ");
        string src = Console.ReadLine();
        Console.Write("Destination: ");
        string dest = Console.ReadLine();
        try {
            File.Copy(src, dest, true);
            Console.WriteLine("Copied successfully!");
        } catch (Exception ex) {
            Console.WriteLine($"Error: {ex.Message}");
        }
    }
}